src/DSL/TSBuilder.cs

```diff
@@ -503,6 +513,16 @@ public void AsyncMethod(string methodName, string returnType, string parameterLi
             Line($"}}");
         }
 
+        public void Method(string methodName, string returnType, string parameterList, Action<TSBlock> methodBodyAction)
+        {
+            Line($"{methodName}({parameterList}): {returnType} {{");
+            Indent(() =>
+            {
+                methodBodyAction.Invoke(new TSBlock(this));
+            });
+            Line($"}}");
+        }
+
         public void AsyncMethod(string methodName, string returnType, Action<TSParameterList> parameterListAction, Action<TSBlock> methodBodyAction)
```

Comment:
With this change, are we still using AsyncMethod anywhere? If not, please remove it.